File: frontend/src/types/StopPointAdress.ts
import { StopPoint } from '@/types/StopPoint';
import { Adress } from '@/types/Adress';

export interface StopPointAdress {
  StopPointAdressId: string;
  StopPoint: StopPoint;
  Adress: Adress;
  StartDate: string;
  EndDate: string;
}

File: backend/Models/StoppointAdress.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Flyt.Models
{
    public class StoppointAdress
    {
        public int Id { get; set; }
        public int StoppointId { get; set; }
        public Stoppoint Stoppoint { get; set; }
        public int? AdressId { get; set; }
        public Adress Adress { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
    }
}
